### DISCUSSION TITLE
Poetry add package show error "Unterminated string starting at: line 1 column 10228 (char 10227)"

### DISCUSSION THREAD

**Discussion by gabvillacis** [python-poetry/poetry#8795] [Q&A]
Once I successfully initialize the Poetry project and attempt to add new packages, I encounter the following error:
"Unterminated string starting at: line 1 column 10228 (char 10227)"

This is my pyproject.toml:

[tool.poetry]
name = "prueba"
version = "0.1.0"
description = "bla"
authors = ["Cris"]
readme = "README.md"

[tool.poetry.dependencies]
python = "^3.11"

[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"

And this is the error:

![WhatsApp Image 2023-12-14 at 22 09 57_120a0399](https://github.com/python-poetry/poetry/assets/55613545/9c2500f9-ff66-4859-9626-232b1b87c13a)

My environment is: python 3.11.7 y poetry 1.7.1.

Can someone please help me?

**Comment by clintonroy** [comment]
At a guess it's a syntax error in your pyproject.toml. Try running poetry check. If you attach it, only as text.

**Reply by cristianhrc** [reply]
Here you are. Appreciate any additional comment. 

[tool.poetry]
name = "prueba"
version = "0.1.0"
description = "bla"
authors = ["Cris"]
readme = "README.md"

[tool.poetry.dependencies]
python = "^3.11"

[build-system]
requires = ["poetry-core"]
build-backend = "poetry.core.masonry.api"

**Comment by clintonroy** [comment]
my guess is that it's something funky in your cache, try cleaning it out.

poetry cache clear . --all

I guess it could also be some funky http proxy requiring you to login?
